tests/legacy/test-functions.sh

```
@@ -0,0 +1,375 @@
+
+test_warning() {
+    echo -e "${YELLOW}⚠️  $1${NC}"
+}
+
+test_error() {
+    echo -e "${RED}❌ $1${NC}"
+}
+
+# Function to run a single test
+run_single_test() {
+    local test_name="$1"
```

Shellcheck (0.10.0)

[warning] test_name appears unused. Verify use (or export if used externally).

(SC2034)